README.md

```diff
@@ -289,3 +289,39 @@ walletConnect:
 ```
 
 Alternatively you can set the bridge by declaring the env var `WALLET_CONNECT_BRIDGE` that would take precedence over the config.
+
+### transactions
+
+Transactions config. Used to apply some postprocessing toevery transaction submitted through the interface
+
+**Config format:**
+
+```yaml
+transactions:
+  sentinel: string
+```
+
+Where:
+
+- `sentinel` is lowercase hex value appended to transaction input data and followed by on-chain analytics flags
+
+Sentinel can also be passed as an environment variable, e.g. `SENTINEL=4e3a yarn start`. In which case it overrides the `sentinel` from config file.
+
+Following the `sentinel` are 6 digits: 2 for provider name, 1 for mobile or desktop flag, 2 for browser name, 1 for screen size.
```

Comment:
Can you link to the line of code per list, to know the values. Or add a new README with them. If someone wants to understand a value, it would be great to be able to know how to do it. It's even good to add an example of a transaction